crates/rspack_plugin_javascript/src/plugin/impl_plugin_for_js_plugin.rs

```diff
@@ -245,6 +245,11 @@ impl Plugin for JsPlugin {
         ..Default::default()
       };
 
+      let format = JsMinifyFormatOptions {
+        ascii_only: minify_options.ascii_only.is_some(),
```

Comment:
Should be using `unwrap_or_default`, since there might be `Some(false)` in this case